packages/backend/src/database/entities/passwordLogins.ts

```diff
@@ -22,3 +24,16 @@ export const createPasswordLogin = async (
         passwordLoginIn,
     );
 };
+
+export const updatePassword = async (
+    userId: number,
+    newPassword: string,
+): Promise<void> => {
+    await database<DbPasswordLogin>('password_logins')
+        .where({
+            user_id: userId,
+        })
+        .update({
+            password_hash: bcrypt.hashSync(newPassword, bcrypt.genSaltSync()),
```

Comment:
btw why not use bcrypt async? It's going to block all requests while generating the salt